README.md

```diff
@@ -125,168 +105,144 @@ _Source:_ [What is Docker](https://www.docker.com/what-docker)
+* [Rescoyl](https://github.com/noteed/rescoyl) - Private Docker registry (free and open source) by [@noteed][noteed]
 * [Quay.io](https://quay.io/) (part of CoreOS) - Secure hosting for private Docker repositories
 * [TreeScale](https://treescale.com/) - Build and Distriubute container based applications.
+* [VMWare Harbor](http://vmware.github.io/harbor/) Project Harbor by VMWare is an enterprise-class registry server that stores and distributes Docker images. Harbor extends the open source Docker Distribution by adding the functionalities usually required by an enterprise, such as security, identity and management.
 
-## Hosting for containers
-* [Amazon ECS](http://aws.amazon.com/ecs/) - A management service on EC2 that supports Docker containers.
-* [ContainerShip Cloud][containership] - Multi-Cloud Container Hosting Automation Platform.
-* [Docker Cloud](https://cloud.docker.com/) - Former Tutum
-* [Giant Swarm](https://giantswarm.io/) - Simple microservice infrastructure. Deploy your containers in seconds.
-* [Google Container Engine](https://cloud.google.com/container-engine/docs/) - Docker containers on Google Cloud Computing powered by [Kubernetes][kubernetes].
-* [Hyper_](https://hyper.sh/) - Secure container hosting service with "nano-containers" and per-second billing.
-* [IBM Bluemix](https://console.ng.bluemix.net/) - Run Docker containers in a hosted cloud environment on IBM Bluemix.
-* [OpenShift Dedicated](https://www.openshift.com/dedicated/index.html) - A hosted [OpenShift][openshift] cluster for running your Docker containers managed by Red Hat.
-* [Orchard](https://www.orchardup.com/) (part of Docker Inc) - Get a Docker host in the cloud, instantly.
-* [Triton](https://www.joyent.com/) - Elastic container-native infrastructure by Joyent.
+## Images
+* [Base Image](https://github.com/phusion/baseimage-docker) by [@phusion](https://github.com/phusion/)
+* [Busybox](https://github.com/jpetazzo/docker-busybox) - with either `buildroot` or Ubuntu's `busybox-static` by [@jpetazzo][jpetazzo]
+* [chaperone-docker](https://github.com/garywiz/chaperone-docker) - A set of images using the Chaperone process manager, including a lean Alpine image, LAMP, LEMP, and bare-bones base kits.
+* [docker-alpine][alpine] - A super small Docker base image *(5MB)* using Alpine Linux by [@gliderlabs][gliderlabs]
+* [docker-fluentd][fluentd] - the Container to Log Other Containers' Logs by [@kiyoto][kiyoto]
+* [nvidia-docker](https://github.com/NVIDIA/nvidia-docker) - Build and run Docker containers leveraging NVIDIA GPUs.
+* [Official Images from Docker Hub](https://github.com/docker-library/official-images)
+* [OpenWRT](http://www.zoobab.com/docker-openwrt-image) by [@zoobab](https://github.com/zoobab)
+* [passenger-docker](https://github.com/phusion/passenger-docker) - Docker base images for Ruby, Python, Node.js and Meteor web apps by [@phusion](https://github.com/phusion)
```

Comment:
Here we use _passenger-docker_ like the repo name, but sometimes we kind of beautify such names to e.g. _Base Image_ instead of _baseimage-docker_ (see above). Do we have some kind of rule? I tend to stick to the repo names, otherwise it becomes more likely that names clash.